sw/device/silicon_creator/mask_rom/mask_rom.c

```diff
@@ -213,9 +214,32 @@ static rom_error_t mask_rom_boot(const manifest_t *manifest,
   sec_mmio_check_values(rnd_uint32());
   sec_mmio_check_counters(/*expected_check_count=*/3);
 
+// Configure address translation, compute the epmp regions and the entry 
```

Comment:
Related to my comment above I think it would be good to pass around a hardened enum specifying which slot the manifest/rom_ext is in rather than trying to extract address information from the manifest.